engine/src/androidTest/java/com/google/android/fhir/db/impl/ResourceDatabaseMigrationTest.kt

```diff
@@ -205,24 +210,29 @@ class ResourceDatabaseMigrationTest {
     val retrievedTask: String?
     val localChangeEntityTimeStamp: Long
     val resourceEntityLastUpdatedLocal: Long
+    val localChangeEntityCorruptedTimeStamp: Long
+
     getMigratedRoomDatabase().apply {
       retrievedTask = this.resourceDao().getResource(taskId, ResourceType.Task)
       resourceEntityLastUpdatedLocal =
         query("Select lastUpdatedLocal from ResourceEntity", null).let {
           it.moveToFirst()
           it.getLong(0)
         }
-      localChangeEntityTimeStamp =
-        query("Select timestamp from LocalChangeEntity", null).let {
-          it.moveToFirst()
-          it.getLong(0)
-        }
+
+      query("Select timestamp from LocalChangeEntity", null).let {
```

Comment:
nit: capitalise SQL keywords
```suggestion
      query("SELECT timestamp FROM LocalChangeEntity", null).let {
```